### ISSUE TITLE
[Bug Report] normal text opacity changed in md2

### ISSUE THREAD

**Issue by 185driver** [vuetifyjs/vuetify#9366]
### Environment
**Vuetify Version:** 2.1.3
**Vue Version:** 2.6.10
**Browsers:** Chrome 78.0.3904.50
**OS:** Windows 10

### Steps to reproduce
Note the text color of `v-card-text` components in the browser's dev-tools.

### Expected Behavior
The normal text color should be `#00000099`.

### Actual Behavior
The normal text color is instead `#0000008a`. This results in a reduced contrast of approximately 4.61 rather than approximately 5.74.

### Reproduction Link
<a href="https://codepen.io/185driver/pen/rNNLNZG" target="_blank">https://codepen.io/185driver/pen/rNNLNZG</a>

### Other comments
Google discusses the use of 60% opacity (instead of the older 54% opacity) for normal text in various places:
https://material.io/design/color/text-legibility.html#text-backgrounds
https://material.io/components/dialogs/#specs
https://material.io/components/cards/#specs

It's a small change, but it does help a bit, and appears to be the new spec. There may also be other components affected. My observation so far has been related to cards and dialogs.

**Comment by venikman** [comment]
I will try to do it on the weekend if no one doesn't mind.

**Comment by johnleider** [comment]
Now that's some attention to detail, good find @185driver
